src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double baseUpper;
+    private double baseLower;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double baseUpper, double baseLower, double height) {
+        super(color);
+        this.baseUpper = baseUpper;
+        this.baseLower = baseLower;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((baseUpper + baseLower) / 2) * height;
+    }
+
+    @Override
```

Comment:
The method 'draw' should not start with an empty line according to the checklist.